ddtrace/tracer/span.go

```diff
@@ -170,6 +170,24 @@ func (s *span) setSamplingPriority(priority int, sampler samplernames.SamplerNam
 	s.setSamplingPriorityLocked(priority, sampler)
 }
 
+// LocalRootSpan returns the root span of the span's trace.
+func (s *span) LocalRootSpan() Span {
```

Comment:
Just a heads up that likely in v2 we want to stop relying on a public interface that we then implement with our own internal struct type, since this forces us to never expand our API without requiring a major version bump. We should think about how this kind of function could affect that kind of change, and if we would want to create some other way for appsec to retrieve this Span (or just set these tags directly)